packages/suite/src/views/settings/SettingsDebug/Passwords.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import styled from 'styled-components';
+import TrezorConnect from '@trezor/connect';
+import crypto from 'crypto';
+import { Button } from '@trezor/components';
+
+import { SectionItem, TextColumn, ActionColumn } from 'src/components/suite';
+import { useSelector, useActions } from 'src/hooks/suite';
+import * as metadataActions from 'src/actions/suite/metadataActions';
+import * as metadataUtils from 'src/utils/suite/metadata';
+import type { PasswordEntry } from 'src/types/suite/metadata';
+import { METADATA } from 'src/actions/suite/constants';
+import { selectSelectedProviderForPasswords } from 'src/reducers/suite/metadataReducer';
+
+const PasswordsList = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    flex: 1;
+`;
+
+const PasswordEntryRow = styled.div`
+    margin-bottom: 4px;
+`;
+
+const PasswordEntryBody = styled.div`
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    margin-left: 8px;
+`;
+
+const PasswordEntryTitle = styled.div`
+    font-size: 14px;
+`;
+
+const PasswordEntryUsername = styled.div`
+    font-size: 14px;
+`;
+
+const PasswordEntryPassword = styled.div`
+    font-size: 14px;
+`;
+
+const FILENAME_MESS = '5f91add3fa1c3c76e90c90a3bd0999e2bd7833d06a483fe884ee60397aca277a';
+const HD_HARDENED = 0x80000000;
+// eslint-disable-next-line no-bitwise
+const PATH = [(10016 | HD_HARDENED) >>> 0, 0];
+const DEFAULT_KEYPHRASE = 'Activate TREZOR Password Manager?';
+const DEFAULT_NONCE =
+    '2d650551248d792eabf628f451200d7f51cb63e46aadcbb1038aacb05e8c8aee2d650551248d792eabf628f451200d7f51cb63e46aadcbb1038aacb05e8c8aee';
+
+interface PasswordEntryProps extends PasswordEntry {
+    devicePath: string;
+}
+
+const getDisplayKey = (title: string, username: string) =>
+    // title = this.bgStore.isUrl(title) ? this.bgStore.decomposeUrl(title).domain : title;
```

Comment:
Remove this comment?

Reply:
f4fd7e0a811e1eec053b5e9fc046e9a3da258a87